lib/active_merchant/billing/gateways/decidir_plus.rb

```diff
@@ -21,6 +21,7 @@ def purchase(money, payment, options = {})
 
         add_payment(post, payment, options)
         add_purchase_data(post, money, payment, options)
+        add_fraud_detection(post, options)
```

Comment:
This needs to be done for auth/purchase I believe

Reply:
Oh wait this is Phase 1 so only purchase ignore meeee

Reply:
Yea, I am just implementing the method to add `fraud_detection`. This should allow Jawad to simply call the method when he implements the `authorize` transaction method for the gateway.

Reply:
love to see this lively discussion and teamwork in action 🤝